components/MDXComponents.tsx

```diff
@@ -25,7 +25,7 @@ const OffsetBox = DS.styled('div', {
 });
 
 const LinkHeading = (props) => (
-  <DS.Text {...props} css={{ ...props.css, scrollMarginTop: DS.theme.space.$9 }}>
+  <DS.Text {...props} css={{ ...props.css }}>
     <NextLink href={`#${props.id}`} passHref>
       <DS.Box
         as="a"
```

Comment:
```suggestion
      <DS.Box
        href={`#${props.id}`}
        as="a"
```
No need for NextLink here, can pass it to the `<a>` itself (since there's no client side navigation)

Reply:
Good point, I'll change that!